script/repl

```diff
@@ -8,4 +8,4 @@ cd "$(dirname "$0")/.."
 
 cabal v2-build all --only-dependencies
 
-cabal v2-exec env -- -u GHC_ENVIRONMENT ghci -ghci-script=.ghci.repl $(script/ghci-flags) -no-ignore-dot-ghci $@
+cabal v2-exec env -- -u GHC_ENVIRONMENT ghci -ghci-script=.ghci.repl $(script/ghci-flags) -no-ignore-dot-ghci -fexternal-interpreter -package pretty-simple $@
```

Comment:
Running on the background thread requires `-fexternal-interpreter` for it to be at all efficient. `-fexternal-interpreter` disables `-package`, so I’ve added this flag to make sure that my ghci conf works.